sdk/bpf/dependencies/.crates2.json

```diff
@@ -0,0 +1 @@
+{"installs":{"xargo 0.3.22 (registry+https://github.com/rust-lang/crates.io-index)":{"version_req":"0.3.22","bins":["xargo","xargo-check"],"features":[],"all_features":false,"no_default_features":false,"profile":"release","target":"x86_64-apple-darwin","rustc":"rustc 1.45.1 (c367798cf 2020-07-26)\nbinary: rustc\ncommit-hash: c367798cfd3817ca6ae908ce675d1d99242af148\ncommit-date: 2020-07-26\nhost: x86_64-apple-darwin\nrelease: 1.45.1\nLLVM version: 10.0\n"}}}
```

Comment:
Why are we commiting the xargo binaries?

Reply:
Cause I suck.  I'll remove them